web/src/beta/lib/core/Crust/Plugins/api.ts

```diff
@@ -540,6 +542,7 @@ export function commonReearth({
         flyToGround,
         setView,
         flyToBBox,
+        rotateCameraOnCenter,
```

Comment:
Since it's already a method of `camera`, naming as `rotateOnCenter` will be better?